test/integration/ResponseValidationTest.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const TestHelper = require('../TestHelper');
+const mockyeah = TestHelper.mockyeah;
+const request = TestHelper.request;
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('Response Validation', () => {
+  it('should validate response option(s) are correct', done => {
+    try {
+      // Use incorrect option 'file'
+      mockyeah.get('/some/service/end/point', { file: './fixtures/some-data.json' });
+    } catch (err) {
+      expect(err.message)
+      .to.equal('Response option(s) invalid');
```

Comment:
The tabbing on this line should be fixed up.  It could be bumped up to the previous line as so: `expect(...).to.equal(...)`.